cmd/di.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/mysteriumnetwork/node/core/connection"
 	"github.com/mysteriumnetwork/node/core/ip"
 	"github.com/mysteriumnetwork/node/core/location"
+	location_factory "github.com/mysteriumnetwork/node/core/location/factory"
```

Comment:
I don't know about this aliasing and 'factory'. Previous version was rather straightforward: `location.CreateLocationResolver`. Perhaps `location.CreateResolver` would be even better? What do we actually gain here from moving DI to a separate sub-package?

Reply:
As mentioned before, to avoid loops between package dependencies: `di` -> `location` -> `di`